src/main/java/lotto/domain/Budget.java

```diff
@@ -0,0 +1,18 @@
+package lotto.domain;
+
+public class Budget {
+
+    private int value;
+
+    public Budget(int budget) {
+        this.value = budget;
+    }
+
+    public int getValue() {
+        return value;
+    }
+
+    public int canBuyQuantity() {
+        return value / Lotto.LOTTO_PRICE;
+    }
```

Comment:
```suggestion
    public int canBuyQuantity() {
        return value / Lotto.LOTTO_PRICE;
    }
```

우리 `Budget`을 그냥 숫자로가 바로볼까요~(`Budget` 이름이 잘못되었다는것은 아닙니다, 이름변경에 대한 코멘트 아닙니다!)
`Number`이름으로 생각하고 다시 바라볼께요.

`Number.canBuyQuantity` "숫자가 살수있는 수량을 계산한다?"  뭔가 이상합니다.
적절한 책임인가?

그럼 다시 `Number`에게 어떤 메시지를 시켜볼지 고민해보면.
`Number.divide(..)`
`Budget.divide(..)`
 저는 자연스럽네요 이게!

그렇다면, Budget 이 LOTTO_PRICE를 가지는게 자연스러운가에 대한 고민을 좀 더 다른방향으로 풀어 볼수있을까요?
네이밍이란건 정말 중요합니다. 우리의 생각을 복잡하게 만들수도 있고, 때론 본질을 보지 못하게 가로막을수도 있다고 생각해요~ 😄